packages/common/utils/src/array/isSubsest.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { isSubset } from '.';
+
+describe('isSubset', () => {
+  it('should return true if the first array is empty', () => {
+    // Given
+    const value1 = [] as string[];
+    const value2 = [1, 2, 3];
+
+    // When
+    const result = isSubset(value1, value2);
+
+    // Then
+    expect(result).toBe(true);
+  });
```

Comment:
```suggestion
  it('should return true if the first array is empty', () => {
    // Given
    const subset = [] as string[];
    const wholeSet = [1, 2, 3];

    // When
    const result = isSubset(subset, wholeSet);

    // Then
    expect(result).toBe(true);
  });
```

Reply:
@okinawaa 
thank you for your review! 🙏

i understand your suggestion.
so i will fix to parameter name more exactly 🛠️